src/ui/ServerAdministration/UsersTable/UserSidebar/UserStatusRoleTable.tsx

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2018 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import * as React from 'react';
+
+import {
+  ButtonDropdown,
+  DropdownItem,
+  DropdownMenu,
+  DropdownToggle,
+} from 'reactstrap';
+import { ISelection } from '..';
+import * as ros from '../../../../services/ros';
+
+export interface IUserStatusRoleTableProps {
+  selection: ISelection;
+  roleDropdownOpen: boolean;
+  toggleRoleDropdown: () => void;
+  onRoleChanged: (role: ros.UserRole) => void;
+  onStatusChanged: (status: ros.UserStatus) => void;
+  statusDropdownOpen: boolean;
+  toggleStatusDropdown: () => void;
+}
+
+export const UserStatusRoleTable = ({
```

Comment:
I think there is something off with the white spacing in this component, specifically
- between the two dropdown buttons - perhaps these should be `size="sm"`?
- between the "status" label and the dropdown
- because its only two dropdown buttons the tabular layout feels like the button to select the role is a button being horizontally centered a bit off. I am a little troubled by this - I don't have a good idea as to what would fix that 😕 ... and perhaps that should not hold back this PR, but we need to find a good solution for this as I imagine we might want to use this pattern elsewhere.

Reply:
Yes, I didn't particularly like the layout of these buttons, but while my React skills are top notch 😎, my css skills are somewhat lacking 😓 If you want to give it 10-15 minutes and try to arrange it nicer, I wouldn't mind, but as you said - it's not such a big deal, and although a little ugly, it does get the job done.